### PR TITLE
EPBDS-13317 and EPBDS-13274

### PR DESCRIPTION
**Pull request by AlexSamBY**
depends on EPBDS-13274

### CODE REVIEW

File: repos/web/.env.production [hunk 1]
```diff
@@ -9,7 +9,7 @@ REACT_APP_RELEASE_VERSION=1.0
 REACT_APP_API_REQUEST_TIMEOUT=60000
 REACT_APP_SOCKET_CONNECT_TIMEOUT=60000
 # When API_URL is empty, defaults to current site
-REACT_APP_API_URL=
+REACT_APP_API_URL=ttps://openl-rating-app-nightly.genci0.eisgroup.com/
```
Comment (yurkom) [inline repos/web/.env.production]: https:

File: repos/web/.env.development [hunk 2]
```diff
@@ -8,7 +8,7 @@ REACT_APP_RELEASE_VERSION=1.0
 REACT_APP_API_REQUEST_TIMEOUT=5000
 REACT_APP_SOCKET_CONNECT_TIMEOUT=5000
 # When API_URL is empty, defaults to current site
-REACT_APP_API_URL=http://localhost:8080
+REACT_APP_API_URL=https://openl-rating-app-nightly.genci0.eisgroup.com/
```
Comment (yurkom) [inline repos/web/.env.development]: Let's keep localhost for development

File: repos/web/.env.production (hunk 1 above)
Comment (AlexSamBY) [reply repos/web/.env.production]: resolved

File: repos/web/.env.development (hunk 2 above)
Comment (AlexSamBY) [reply repos/web/.env.development]: done

**Review by yurkom — APPROVED** [review]

**Pull request merged**